packages/types/src/element/element.ts

```diff
@@ -52,11 +61,11 @@ export interface BorderRadius {
   locked: boolean;
 }
 
+export type ElementId = string;
+
 export interface Element extends ElementBox {
-  id: string;
+  id: ElementId;
   type: string;
-  isBackground: boolean;
```

Comment:
I moved this to `ShapeElement` and `MediaElement` since only these two can ever really background elements. However, this rightfully causes a TSC error for the `getBox()` function in the `units` package:

```
packages/units/src/dimensions.ts:144:41 - error TS2339: Property 'isBackground' does not exist on type 'Element'.

144   { x, y, width, height, rotationAngle, isBackground }: Element,
                                            ~~~~~~~~~~~~
```

Either that function needs to be updated, or we undo this change.

Any thoughts appreciated.

Reply:
I'm wondering if we should still use a union type for the `Page` `element` prop instead of `Element` as it currently is.

This is just one case where we'll have this issue -- when we get to the `story-editor` package, essentially every time when `element` is used as a prop, it will be a problem that the `Page` type has `elements: Element[]` instead of accepting all element types.
Alternatively we'll need to extend the central `Page` type in every package that uses `element` as a prop with non-shared props.

Reply:
Updated `getBox` in [a8bfd30](https://github.com/GoogleForCreators/web-stories-wp/pull/12429/commits/a8bfd305af3f9d7f32709922c67884c8402b170d).